evm/its.js

```diff
@@ -425,51 +418,52 @@ async function processCommand(config, chain, options) {
             break;
         }
 
-        case 'setTrustedAddress': {
+        case 'set-trusted-address': {
+            const [trustedChain, trustedAddress] = args;
             const owner = await new Contract(interchainTokenService.address, IOwnable.abi, wallet).owner();
 
             if (owner.toLowerCase() !== walletAddress.toLowerCase()) {
                 throw new Error(`${action} can only be performed by contract owner: ${owner}`);
             }
 
-            validateParameters({ isNonEmptyString: { trustedChain: options.trustedChain } });
+            validateParameters({ isNonEmptyString: { trustedChain } });
 
             let trustedChains, trustedAddresses;
 
-            if (options.trustedChain === 'all') {
+            if (trustedChain === 'all') {
                 const itsChains = Object.values(config.chains).filter((chain) => chain.contracts?.InterchainTokenService?.skip !== true);
```

Comment:
`skip` doesn't exist anymore. create a separate PR to refactor this logic to use the new helper methods